ts/components/settings/section/CategoryPrivacy.tsx

```diff
@@ -41,8 +45,11 @@ export const SettingsCategoryPrivacy = (props: {
   hasPassword: boolean | null;
   onPasswordUpdated: (action: string) => void;
 }) => {
+  const [isLinkPreviewsOn, setIsLinkPreviewsOn] = React.useState(
+    Boolean(window.getSettingValue(SettingsKey.settingsLinkPreview))
+  );
+
```

Comment:
almost there.
I thing it should still be working fine if you 
replace that useState with  just 
```ts
const isLinkPreviewsOn = Boolean(window.getSettingValue(SettingsKey.settingsLinkPreview))
```

Reply:
it seems like if I try it without the state, It wont render again and I wont be able to change it everytime. That is why I started using a state in the first place. We can try tomorrow, but I think we might need the state